tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/MinorDottySuite.scala

```diff
@@ -577,4 +577,118 @@ class MinorDottySuite extends BaseDottySuite {
       )
     )
   }
+
+  test("simple-derives") {
+    val derivesEnum = """|enum Tree[T] derives Eq, Ordering, Show {
```

Comment:
modify to `scala.derives.Ordering` to test fully qualified names are also parsed ok

Reply:
Added!